README.md

```diff
@@ -1,20 +1,22 @@
-
 # GitExtensions.GerritPlugin
 
 GitExtensions.GerritPlugin is a plugin for GitExtensions to work with a [Gerrit](https://www.gerritcodereview.com/) as Git server.
 
+
```

Comment:
nit pick - are you using a styleguide for the changes?
This adds double newlines where not needed in my opinion.
Google styleguide is quite simple
https://google.github.io/styleguide/docguide/style.html